docs/docs-content/vm-management/create-manage-vm/using-datavolumes.md

```diff
@@ -0,0 +1,184 @@
+
+<br />
+
+### Importing disk images as templates
+
+The following manifest can be added to a manifest layer in a Palette cluster profile to define a `DataVolume` that imports the example disk template for Ubuntu 22.04 into the `vmo-golden-images` namespace:
+
+```
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: "template-ubuntu-2204"
+  namespace: "vmo-golden-images"
+  annotations:
+    cdi.kubevirt.io/storage.deleteAfterCompletion: "false"
+spec:
+  storage:
+    accessModes:
+      - ReadWriteMany
+    resources:
+      requests:
+        storage: 50Gi
+    volumeMode: Block      # or "Filesystem", depending on the CSI used
+  source:
+    registry:
+      url: "docker://gcr.io/spectro-images-public/release/vm-dashboard/os/ubuntu-container-disk:22.04"
+```
+
+Possible `source` options are described in the [CDI DataVolumes documentation](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/datavolumes.md#source). The example above uses the `registry` source to import the data from a container disk on an external Docker registry.
+
+Note 2 important aspects of this DataVolume:
+* It sets an annotation to prevent deleting the `DataVolume` object after if finishes the import process. This is necessary to prevent Palette from continuously recreating the `DataVolume` object, resulting in error messages.
+* The `volumeMode` needs to align with the mode that the CSI supports for `ReadWriteMany` access. For some CSIs such as Rook-Ceph and EMC PowerMax this is `Block` mode, while for other CSIs such as Portworx, Longhorn and EMC PowerFlex this is `Filesystem` mode.
+
+:::info
+When `Filesystem` mode is selected, the CDI will automatically apply a small overhead factor to the PVC size to account for the storage space lost due to filesystem overhead. As a result, the PVC for such a template will be slightly larger. Any clones made from this PVC also need to use this slightly larger minimum size.
+:::
+
+As soon as the `DataVolume` resource above is applied to the cluster, the CDI will begin importing the data into the target PVC in the `vmo-golden-images` namespace. The target PVC will have the same name as the `DataVolume` object.
+
+<br />
+
+### Creating vmTemplates with CSI-assisted cloning
+
+The next step is to create a `vmTemplate` resource that clones the PVC in the `vmo-golden-images` namespace. This is done through the `dataVolumeTemplates` section:
+
+```
+apiVersion: spectrocloud.com/v1
+kind: VmTemplate
+metadata:
+  name: ubuntu-2204
+spec:
+  dataVolumeTemplates:
+    - metadata:
+        name: ubuntu-2204
+      spec:
+        source:
+          pvc: 
+            name: template-ubuntu-2204
+            namespace: vmo-golden-images
+        pvc:
+          accessModes:
+            - ReadWriteMany
+          resources:
+            requests:
+              storage: 50Gi
+          volumeMode: Block      # or "Filesystem", depending on the CSI used
+  template:
+    < rest of template >
```

Comment:
Please provide the rest of the template so that learners can copy this. The code preview box will limit how many lines will be displayed and create a scrollbar.